GPU/Common/TextureCacheCommon.cpp

```diff
@@ -804,7 +804,11 @@ void TextureCacheCommon::NotifyFramebuffer(u32 address, VirtualFramebuffer *fram
 					depth ? "DEPTH" : "COLOR", (int)candidates.size(), addr, framebuffer->width, framebuffer->height, depth ? framebuffer->z_stride : framebuffer->fb_stride, GeBufferFormatToString(framebuffer->format));
 			}
 
-			AttachBestCandidate(candidates);
+			// There can actually be multiple ones to update here! This can be the case where two textures point to different framebuffers that share depth buffers.
+			// So we have no choice but to run all the candidate matches.
+			for (int i = 0; i < (int)candidates.size(); i++) {
+				ApplyFramebufferMatch(candidates[i].match, candidates[i].entry, framebuffer->fb_address, framebuffer, candidates[i].channel);
```

Comment:
Hmm, it'd be nice to still apply the relevancy logic right?  Really, we want a list of candidates per entry...

-[Unknown]

Reply:
Well here there is only one candidate framebuffer, the framebuffer being notified - we don't care about any other framebuffers here, just if it should be attached to any texture we're already tracking. Just might be multiple of those...

So I don't think the relevancy logic is actually very useful here.